packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -854,6 +888,73 @@ describe('State Machine handler tests', () => {
     expect(call['callStateMachine'].state.value).toBe('S_SEND_CALL_DISCONNECT');
   });
 
+  it('outgoing call with connect after setup, media established before connect ', async () => {
+    const statusPayload = <WebexRequestPayload>(<unknown>{
+      statusCode: 200,
+      body: mockStatusBody,
+    });
+    const dummyEvent = {
+      type: 'E_SEND_CALL_SETUP',
+      data: undefined as any,
+    };
+
+    const postMediaSpy = jest.spyOn(call as any, 'postMedia');
+
+    webex.request.mockReturnValue(statusPayload);
+
+    call.sendCallStateMachineEvt(dummyEvent as CallEvent);
+    expect(call['callStateMachine'].state.value).toBe('S_SEND_CALL_SETUP');
+    // dummyEvent.type = 'E_RECV_CALL_PROGRESS';
+    // call.sendCallStateMachineEvt(dummyEvent as CallEvent);
+    // expect(call['callStateMachine'].state.value).toBe('S_RECV_CALL_PROGRESS');
+
+    dummyEvent.type = 'E_SEND_ROAP_OFFER';
```

Comment:
Creating a separate object would be cleaner, if any change is needed later it may mess up the code

Reply:
I agree. I've done it for this test which I've added now. Other tests can be taken care of in the future.